Netkan/Services/CachingHttpService.cs

```diff
@@ -20,8 +21,32 @@ public CachingHttpService(NetFileCache cache, bool overwrite = false)
             _overwriteCache = overwrite;
         }
 
-        public string DownloadPackage(Uri url, string identifier, DateTime? updated)
+        public string DownloadModule(Metadata metadata)
         {
+            try
+            {
+                return DownloadPackage(metadata.Download, metadata.Identifier, metadata.RemoteTimestamp);
+            }
+            catch (Exception exc)
+            {
+                var fallback = metadata.FallbackDownload;
+                if (fallback == null)
+                {
+                    throw;
+                }
+                else
+                {
+                    return DownloadPackage(fallback, metadata.Identifier, metadata.RemoteTimestamp, metadata.Download);
```

Comment:
A log statement would probably be handy here, so that you can see that the fallback is used.
I would go for a `Log.Info` at least.

Reply:
Good idea, borrowed the one from Core.